ios/Codeforces Watcher/Application/core/design/Font.swift

```diff
@@ -27,4 +27,6 @@ public class Font {
         UIFont(name: "Roboto-Regular", size: 11) ?? UIFont.systemFont(ofSize: 11)
     public static let textHintBold =
         UIFont(name: "Roboto-Medium", size: 11) ?? UIFont.systemFont(ofSize: 11)
+    public static let textField =
```

Comment:
Not the best name for the font 😄 Where is used aside from `TextField`?

Reply:
Only there, I'm not sure about adding this here, but for better readability I did it.
I have set the name from other examples in this file😄
Also, I had a problem, because of the name of this extension, it is not the best way to create the same file with fonts using swiftUI.Font to use it in SwiftUI Views.